modules/theora/video_stream_theora.cpp

```diff
@@ -248,10 +257,9 @@ void VideoStreamPlaybackTheora::set_file(const String &p_file) {
 #pragma warning(disable : 4706)
 #endif
 
-		int ret;
-
 		/* look for further theora headers */
-		while (theora_p && (theora_p < 3) && (ret = ogg_stream_packetout(&to, &op))) {
+		int ret = ogg_stream_packetout(&to, &op);
+		while (theora_p && theora_p < 3 && ret) {
```

Comment:
You can remove the above `#pragma warning(disable : 4706)` I added recently then.